backend/pom.xml

```diff
@@ -3,12 +3,12 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>io.debezium.uipoc</groupId>
-    <artifactId>debezium-uipoc-parent</artifactId>
+    <groupId>io.debezium.ui</groupId>
```

Comment:
@gunnarmorling should we align that groupId to the package namespace of the Java application?

Reply:
Or maybe even io.debezium, like everything else? I don't think it needs a separate group id actually.

Reply:
Makes sense. Let's do that @mdrillin